shells/pipes-shell/node/node.js

```diff
@@ -18,23 +18,23 @@ import {version, paths, storage, test} from './config.js';
 //import {DevtoolsSupport} from '../../lib/devtools-support.js';
 
 // dependencies
-import {RamSlotComposer} from '../../lib/components/ram-slot-composer.js';
```

Comment:
I'd like to ask you to coordinate with me before modifying pipes-shell. 

I'm also actively working on it, and I think now we are in conflict. I'll have to study to figure out how much conflict.

Reply:
I reverted most of the pipes-shell changes, PTAL